hook!-full-game/src/singletons/Defaults.gd

```diff
@@ -0,0 +1,9 @@
+"""
+Holds default constants that are used in several parts of the 
+game's code
+"""
+extends Node
+
+const DEATH_ANIMATION_TIME := 2.5
```

Comment:
Avoid singletons. The player's death animation length shouldn't be a global property.